src/view/screens/Search/Search.tsx

```diff
@@ -1058,7 +1058,10 @@ function SearchHistory({
                     asAnchor
                     anchorNoUnderline
                     onBeforePress={() => onProfileClick(profile)}
-                    style={styles.profilePressable}>
+                    style={[
+                      styles.profilePressable,
+                      isMobile && styles.profilePressableMobile,
```

Comment:
Is there a downside to make this style unconditional?

Reply:
It's okay to apply style to all screen sizes. I just pushed the change